android/engine/src/main/java/org/smartregister/fhircore/engine/p2p/dao/P2PSenderTransferDao.kt

```diff
@@ -82,7 +83,9 @@ constructor(
         } else {
           highestRecordId
         }
-      Timber.e("Sending ${it.resource.resourceType} with id ====== ${it.resource.logicalId}")
+      Timber.e(
```

Comment:
Any reason we are logging debug information as errors?

Reply:
The log level has been updated to `info`